PROJECT-BRIEF.md

```diff
@@ -4,10 +4,10 @@
 
 ### Locations for things
 
-- Site: https://tcl-47-smart-shopping-list.web.app
+- Site: https://tcl-47-smart-shopping-li-6b6a7.web.app
 - Repo: https://github.com/the-collab-lab/tcl-47-smart-shopping-list
 - Clone URL: https://github.com/the-collab-lab/tcl-47-smart-shopping-list.git
-- Issue list: https://github.com/the-collab-lab/tcl-47-smart-shopping-list/projects/1
+- Issue list: https://github.com/orgs/the-collab-lab/projects/24/views/1
```

Comment:
```suggestion
- Issue list: https://github.com/the-collab-lab/tcl-47-smart-shopping-list/issues
```
Should the issues list actually point to the issues tab?

Reply:
@jendevelops That makes a lot more sense! Thanks!!!